app/uk/gov/hmrc/emailverification/services/AuditService.scala

```diff
@@ -313,7 +313,7 @@ class AuditService @Inject() (
       "statusCode" -> JsString(responseStatus.toString)
     )
     logger.warn(s"[GG-5646] EmailVerification outcome requested for credId $credId. ${requestContextForLog(request)}")
-    sendEventWithJsonDetails("EmailVerificationOutcomeRequest", details, "HMRC Gateway - Email Verification - Send email verification outcome request")
+    sendEventWithJsonDetails("EmailVerificationOutcomeRequest", details, "HMRC Gateway - Email Verification - Return email verification outcome request")
```

Comment:
Sorry, pls change to "HMRC Gateway - Email Verification - Email address not found or expired"